src/ops/align.jl

```diff
@@ -308,3 +308,40 @@ function count_knots(x)
     _is_constant(x) && return constant(1)  # A constant will always have one knot.
     return obtain_node((x,), CountKnots())
 end
+
+struct Skip{T} <: UnaryNodeOp{T}
```

Comment:
This is correct — but it will do an unnecessary amount of work, and might allocate a lot more than needed.

Maybe have a look at the implementation of `prepend` for an idea...